test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/main.py

```diff
@@ -354,6 +355,7 @@ def _check_interpreter(self, powershell=False):
             if not self.text.startswith('#!powershell\n'):
                 self.reporter.error(
                     path=self.object_path,
+                    name='interpreter-line-powershell',
```

Comment:
I think we can remove this check in a separate PR, as the shebang sanity test covers this.